src/nix/flake.cc.orig

```diff
@@ -0,0 +1,1180 @@
+#include "command.hh"
```

Comment:
You included the file by accident, right?

Reply:
Ooops, yes. Thanks, fixed